core/src/components/textarea/textarea.tsx

```diff
@@ -506,8 +506,75 @@ For textareas that do not have a visible label, developers should use "aria-labe
     );
   }
 
+  private renderLabel() {
+    const { label } = this;
+    if (label === undefined) {
+      return;
+    }
+
+    return (
+      <div class="label-text-wrapper">
+        <div class="label-text">{this.label}</div>
+      </div>
+    );
+  }
+
+  /**
+   * Renders the border container when fill="outline".
+   */
+  private renderLabelContainer() {
+    return this.renderLabel();
+  }
+
   private renderTextarea() {
-    return <Host>Stubbed textarea</Host>;
+    const { inputId, disabled } = this;
+    const mode = getIonMode(this);
+    const value = this.getValue();
+
+    return (
+      <Host
+        aria-disabled={disabled ? 'true' : null}
+        class={createColorClasses(this.color, {
+          [mode]: true,
+          'has-value': this.hasValue(),
+          'has-focus': this.hasFocus,
+        })}
+      >
+        <label class="textarea-wrapper">
+          {this.renderLabelContainer()}
+          <div class="native-wrapper" ref={(el) => (this.textareaWrapper = el)}>
```

Comment:
I will likely need to revisit the `textareaWrapper` element ref in future work with FW-2764 when  working on auto grow. Now that we have more element containers, it might need to become just an `autogrowWrapper` or something, so as not to be confusing with the "textarea-wrapper".